diff --git a/workloads/Kotlin-compose-wasm/build/compose-benchmarks-benchmarks.uninstantiated.mjs b/workloads/Kotlin-compose-wasm/build/compose-benchmarks-benchmarks.uninstantiated.mjs
index 3427ca2..882308d 100644
--- a/workloads/Kotlin-compose-wasm/build/compose-benchmarks-benchmarks.uninstantiated.mjs
+++ b/workloads/Kotlin-compose-wasm/build/compose-benchmarks-benchmarks.uninstantiated.mjs
@@ -383,7 +383,7 @@ export async function instantiate(imports={}, runInitializer=true) {
 
     const wasmFilePath = './compose-benchmarks-benchmarks.wasm';
 
-    const wasmTag = WebAssembly.JSTag ?? new WebAssembly.Tag({ parameters: ['externref'] });
+    const wasmTag = new WebAssembly.Tag({ parameters: ['externref'] });
 
     const importObject = {
         js_code,
